sophia-vote-contract.md

```diff
@@ -100,37 +106,37 @@ contract Vote =
    record state = {
       votes: map(address, candidates) }
 
-   public stateful function init() : state = {
+   stateful entrypoint init() : state = {
       votes = { } }
 
-   public stateful function vote(candidate: address) : bool =
+   stateful entrypoint vote(candidate: address) : bool =
       is_candidate(candidate)
       let new_votes_state = Call.caller :: state.votes[candidate].voters
       put(state{
          votes[candidate].voters = new_votes_state })
       true
 
-   public function count_votes(candidate : address) : int =
+   entrypoint count_votes(candidate : address) : int =
       size(state.votes[candidate].voters)  
 
-   private function size(l : list('a)) : int = size'(l, 0)
+   function size(l : list('a)) : int = size'(l, 0)
    
-   private function size'(l : list('a), x : int) : int =
+   function size'(l : list('a), x : int) : int =
       switch(l)
          [] => x
          _ :: l' => size'(l', x + 1)
 
-   public stateful function add_candidate(candidate: address) : bool =
+   stateful entrypoint add_candidate(candidate: address) : bool =
```

Comment:
I don't see the point in always returning `true`?

Reply:
This is my first time using a language without a return statement. I think it's a reaction to the weird feeling 🙈. Thanks, I have removed them.

Reply:
I have implemented the requested changes. Can you resolve the conversation?